node_helper.js

```diff
@@ -345,11 +345,41 @@ module.exports = NodeHelper.create({
         if (bytes.r[0].test(data) && bytes.r[1].test(data)) {
             this.sendSocketNotification('BYTES', bytes.a);
         } else if (/(WAKE)/g.test(data) && /(UP)/g.test(data)) {
-            exec('/opt/vc/bin/tvservice -p && sudo chvt 6 && sudo chvt 7', null);
-            this.hdmi = true;
+						switch(this.config.mode.toLowerCase()){
```

Comment:
please use uppercase as in the readme, also replace switch with if/elseif

Reply:
no.. the options are case insensitive.. i don't like forcing users to be specific.. users fail.. my job is to NOT let it fail..

Reply:
i hate if/else when there are more than 2 choices..    long time programmer,  make things clear..

Reply:
it's good if you want to catch errors from users, but to have consistency for developers and users, you should do touppercase instead of tolowercase

Reply:
done